interfaces/builtin/acrn.go

```diff
@@ -0,0 +1,72 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2021 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"fmt"
+	"io/ioutil"
+	"regexp"
+	"strings"
+
+	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/interfaces/kmod"
+	"github.com/snapcore/snapd/logger"
+	"github.com/snapcore/snapd/strutil"
+    "github.com/snapcore/snapd/interfaces/apparmor"
+	"github.com/snapcore/snapd/snap"
+)
+
+const acrnSummary = `allows access to the acrn_hsm device`
```

Comment:
So as far as I understand this allows to operate as ACRN Service VM? https://projectacrn.github.io/latest/developer-guides/hld/hld-overview.html#i-o-emulation Can you confirm this?

Reply:
Yes, but I also asked the Intel folks to join the conversation and provide any additional insight on their end.

Reply:
This is correct, this is what the Device Model uses to communicate with the hypervisor, you can read more about it here:
* https://projectacrn.github.io/latest/developer-guides/hld/hld-devicemodel.html#hsm
* https://www.kernel.org/doc/html/latest/virt/acrn/index.html

Reply:
something more high-level like what we use for kurbenetes/docker-support is probably preferable here:

`allows operating as ...`